File: 3168. Minimum Number of Chairs in a Waiting Room.java
class Solution {
    public int minimumChairs(String s) {
        int answer = 0;
        int count = 0;
        for (char c : s.toCharArray()) {
            if (c == 'E') count++;
            else count--;
            answer = Math.max(answer, count);
        }
        return answer;
    }
}

File: 3168. Minimum Number of Chairs in a Waiting Room.go
func minimumChairs(s string) int {
    count := 0
    answer := 0
    for _, i := range s {
        if i == 'E' {
            count += 1
        } else {
            count -= 1
        }
        answer = max(answer, count)
    }
    return answer
}
